canonical-web-and-design/jaas-dashboard
# Swap logo to Juju under Juju

## Done

When deployed in Juju then change the logo to "Juju" from "JAAS"

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The logo should still be "JAAS"
- Change the configuration value `isJuju` in `config.js` to `true` and refresh
- The logo should now be "Juju"

## Details

fixes #355 

## Screenshots

![Screen Shot 2020-04-16 at 1 52 28 PM](https://user-images.githubusercontent.com/532033/79500726-26e77800-7fea-11ea-959c-98fb7c51522e.png)
